examples/python/rippleadd-async.py

```diff
@@ -120,24 +120,27 @@ def unmajority(p, a, b, c):
 qobjs = []
 # Create online (so I/O bound) jobs if we have connetion or local (so CPU bound)
 # jobs otherwise
-if offline == False:
+if offline is False:
```

Comment:
This one is not currently picked by the linter it seems, but could you please change it to:
```
if not offline:
```